### ISSUE TITLE
Branding suggestion: wfID

### ISSUE THREAD

**Issue by SachaG** [web-platform-dx/web-features#1721]
Sorry for dropping this out of the blue, but one of the way I relax is by doing unsolicited brand redesigns instead of my actual work. 

I was thinking about the fact that it's quite hard to search for "web features" as it's a pretty generic term. It's also not very memorable for the same reason. 

My proposal would be to rebrand this project as `wfID`, for `web features ID`. Here's a quick logo idea: 

<img width="800" alt="wfid-logo" src="https://github.com/user-attachments/assets/502fb447-792d-43fb-abf9-45660e61631e">

Why `wfID`?

- it's a more unique name, and thus easier to search for. 
- it lends itself well to becoming an icon or a mark that could appear whenever other sites link to or mention web features.
- it puts the accent on the fact that the project's goal –as I understand it– is to provide canonical IDs for features, as opposed to aggregating and storing metadata about these features (which is a task best left to MDN, CanIUse, State Of…, etc.). 

Anyway as someone external to the project I fully realize I'm not the best qualified person to have an opinion on something as fundamental as its name. I just had the thought pop into my head and thought I'd share my quick logo sketch. But if this doesn't resonate with anybody else feel free to just ignore it!

**Comment by captainbrosset** [comment]
Thanks so much for spending time on this Sacha. I have a few thoughts about this.

I agree that the "web features" phrase is not very searchable/recognizable. I think we didn't really pay too much attention to it initially because we produce a package that others consume. So we're in the background, kind of like BCD. Web-features is the hidden part of the iceberg, while sites like MDN are the visible part, showing the Baseline widget.
I agree, however, that having a memorable name and easy way to search does help, even in our case. Taking the BCD example again, the fact that people call it BCD makes it work kind of like a brand, which helps talk about it.
So I'm on board with finding a short/memorable brand name to refer to this project.

As for the accent on the IDs, I'm not sure I agree. Yes, the IDs are the most important part, which allow cross-references. But, the project also defines:

- Short names and descriptions for features, which provide quick understanding of what the scope of each feature is. These can help both the consumers of our package, as well as the end users of whatever site the package is used on. IDs are not enough on their own to convey what features do.
- Compat data and baseline status, which are very important. In a lot of cases, compat data comes straight from BCD, but not always. And baseline status doesn't exist anywhere else. These, together, constitute a layer a human editorial decisions that provide value to web developers.

I feel like a brand name should capture this to some extent.

Anyway, not trying to stop this effort, and thank you again for the time spent on this. I really like where this is going and agree we need to be thinking about this more.

**Comment by atopal** [comment]
I've actually run into this problem a few times when I have referred to the web features project. The issue is not that it's non memorable, it's just too generic. Searching for "Web Features" doesn't bring it up anywhere on the first search results page, and you have to spend some time explaining what you are referring to specifically when you say "web features". So, having something a bit more unique would certainly help with that (or we just become successful enough that this is the only thing Web Features can reasonably refer to :))

Identifying web features was not the intended goal of this project, that was just a means to an end to be able to provide availability information in the form of the Baseline status. We just organically ended up in a place where having identifiers for web features turned out to be really, really useful, and we decided recently to lean more heavily on that. So, now we have basically two initiatives jumbled into one: Baseline and Web Features. As Patrick and Francois said, Baseline is the user facing part that is already branded and becoming more and more well known for web developers. 

That leaves Web Features, which is not intended for web developers, it's infrastructure and as such doesn't need a high profile brand, but probably a more specific name. Web Feature IDs might be that, even if it doesn't capture some of the other aspects. 

Thank you Sacha for opening up this conversation, and for the great work on the logo!

**Comment by captainbrosset** [comment]
Let's brainstorm on names for this project then. Having a specific name would definitely help people, starting with us. Many times I've said and heard the phrase "web features" without being sure whether we were talking about this repo or about a feature of the web platform.
Some proposals, including Sacha's one:

* wfID
* WebFeatureIndex
* WebFeatureCatalog

Other possible words to consider using: inventory, hub, navigator, map, registry.
Maybe using the word Baseline in the name can help too? BaselineIndex?

**Comment by SachaG** [comment]
> Other possible words to consider using: inventory, hub, navigator, map, registry.

I would add `-pedia` to the list of possible suffixes.

**Comment by ddbeck** [comment]
I'll think on possible names, but one thing I'd like to put down as a consideration: how easy it is to say. If we do rename, I'd like to make sure we're not making harder to say (unfortunately, wfid fails this test—it's twice as many syllables), or causing us to say unnatural things (unfortunately, I'm often saying "web-features features" which makes me inwardly cringe every time I do it).

**Comment by SachaG** [comment]
Yeah I did also notice that "wfID" is quite a mouthful because of the "w".

**Comment by SachaG** [comment]
Just to keep the discussion going – another idea I just had. 

![wpfi](https://github.com/user-attachments/assets/b335d797-b923-4797-9a05-c4db82fbb65c)

**Comment by atopal** [comment]
Hah, I love it, and _Feature Index_ is growing on me. Agreed with Daniel that whatever we switch to should either be easy to say or have an abbreviation that is easy to say, but I'm not sure there is a way around the W without making it very generic again.

**Comment by ddbeck** [comment]
I like "index" quite a lot—it speaks well to cross-referencing other things and makes me think of `index.html`. I got to thinking of what it might look like if we didn't use "feature" (to avoid "feature features") or "Web" (to avoid double-U). Here's what I've got:

- Platform Index (abbreviates to PI, I imagine there's some nice detective, scientist, or π iconography opportunities here)
- Platform Technology Index (PTI)
- Capability Index (CI, too hard to spell and the abbreviation is confusing, but maybe speaks explains what "features" are a bit)

Using some of the words mentioned:

- Platform Catalog (PC)
- Platform Map (PM)
- Capability Catalog (CC, C2?)

And now some bonus attempts at portmanteaus to avoid being abbreviated at all (all too clever by half yet I still kinda love them):

- Platalog
- Platadex
- Featex (or Feadex)
- Webdex (too confusing with WebDX though)

<details>

<summary>More failed experiments</summary>

I also plugged "web features" into an anagram generator and found some… inspiration:

- Beta Few User
- A Tubes Fewer
- Features Web (of course!)

Bonus anagram: "web platform feature" got me to "Bloatware's FTPer fume."

</details>

**Comment by ddbeck** [comment]
It also occurs to me that we could lean really hard into unpronounceableness: Web Technology Features, better known as WTFs.

**Comment by SachaG** [comment]
By the way, something I noticed is that "Web Platform" could potentially be used as an umbrella brand of sorts – in that Baseline's full name is actually Web Platform Baseline, and then we also already have Web Platform Tests. This is why I also tried including it in that logo.

**Comment by tidoust** [comment]
> * Webdex (too confusing with WebDX though)

For info, we already have a project named [Webdex](https://dontcallmedom.github.io/webdex/) that lists terms defined in specs, derived from Webref extracts.

**Comment by captainbrosset** [comment]
Web Platform Feature Index is pretty nice. Or just Web Feature Index, for shorts. And for even shorter, Web Index, which is as fast to say as BCD.

**Comment by atopal** [comment]
Leaning into Web Platform makes sense to me too. And it's a way to disambiguate where the name would otherwise be too generic. Baseline is the same, you can't use it outside of web dev without adding "Web Platform", but see [here](https://blog.logto.io/css-only-infinite-scroll) for an article where the author just says Baseline 2023 without context and it's still clear what it's referring to.

**Comment by captainbrosset** [comment]
Discussed on the WebDX CG call: we decided to close this. We couldn't find a good replacement, and thought changing it now would cause more harm than good.
